packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -238,151 +238,118 @@
 					"value": 0
 				},
 				{
-					"label": "Dimmer 2 state synchronizes brightness level",
+					"label": "This dimmers current state synchronizes brightness level",
 					"value": 1
 				}
 			]
 		},
 		"22": {
-			"label": "Change [On-Off] bi-stable keys",
+			"label": "Change bi-stable input switch behavior",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 1,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Device changes status on key status change",
+					"label": "Output changes on input switch change",
 					"value": 0
 				},
 				{
-					"label": "Status depends on key status",
+					"label": "Output synced with input switch position",
 					"value": 1
 				}
 			]
 		},
 		"23": {
-			"label": "Double click option",
-			"description": "Double-click set lighting at 100%",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 1,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Disable double click",
-					"value": 0
-				},
-				{
-					"label": "Enable double click",
-					"value": 1
-				}
-			]
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Double click sets brightness to 100%",
+			"defaultValue": 1
 		},
 		"24": {
-			"label": "Command frames sent in 2nd and 3rd assoc group",
-			"description": "Actions not resulting in sending frames to assoc",
+			"label": "Command frames sent in 2nd and 3rd assoc group (S1)",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 16,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "all actions send to association groups",
+					"label": "All actions send to association groups",
 					"value": 0
 				},
 				{
-					"label": "do not send when switching ON (single click)",
+					"label": "Do not send when switching ON (single click)",
 					"value": 1
 				},
 				{
-					"label": "do not send when switching OFF (single click)",
+					"label": "Do not send when switching OFF (single click)",
 					"value": 2
 				},
 				{
 					"label": "Do not send when changing dimming level",
 					"value": 4
 				},
 				{
-					"label": "do not send on double click",
+					"label": "Do not send on double click",
 					"value": 8
 				},
 				{
-					"label": "send 0xFF value on double click",
+					"label": "Send 0xFF value on double click",
 					"value": 16
 				}
 			]
```

Comment:
This looks like it should be possible to combine these options. To model that, we use [partial parameters](https://zwave-js.github.io/node-zwave-js/#/config-files/partial-parameters).

Reply:
I think you are right.